### PR TITLE
fix(gateway): Fixed the access issues on the API Gateway

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 361

### 

# Explain the feature

- Get access to the API has revoked.

## Does this PR introduce a breaking change

No

## Screenshots
N/A

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api-gateway-service/service.ts [hunk 1]
```diff
@@ -75,31 +116,38 @@ const server = http.createServer( app );
...
-      } catch ( err ) {
-        return res.status( 403 ).json( err );
-      }
-    } );
+    return null;
   }
-} );
+};
+
+app.get('/graphql', (req, res, next) => {
```
Comment (deshmukhmayur) [inline packages/api-gateway-service/service.ts:131]: Why is the auth middleware just applied to only GET requests??

Should use `.use` to make sure it gets applied to all kinds of request methods.

```suggestion
app.use('/graphql', (req, res, next) => {
```

Comment (riginoommen) [reply packages/api-gateway-service/service.ts:131]: I made it as get for accessing the playground. if you are using the postman/from portal both way of auth will work

**Comment by riginoommen** [comment]
> @riginoommen What's the process for creating the LocalJWT token? Is it something static that we'll add to the environment?
> If that's the case, then I wouldn't suggest doing that.
> 
> We need a more flexible way to manage the API Keys. So an API endpoint which we can use to create API Keys for specific users as and when needed would be the way to go, at least in my opinion.
> 
> @sayak-sarkar @diwanshi Thoughts?

For issuing the jwtoken you have store the secret jwttoken in the environment configuration. not in the code

**Comment by sayak-sarkar** [comment]
@riginoommen Please follow the team norms for PR Titles!

**Comment by deshmukhmayur** [comment]
@riginoommen Can you rebase it?

**Comment by riginoommen** [comment]
> @riginoommen Can you rebase it?

Rebased

**Pull request merged**